code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -208,3 +208,40 @@ Difficulty: Medium
 	playsound(user, 'sound/magic/staff_change.ogg', 200, 0)
 	A.telegraph()
 	storm_cooldown = world.time + 200
+
+//Crown and Jewel of the Wastes
+/obj/item/clothing/ears/jewel_of_the_wastes //Takes headset but makes you have ear protection + same faction as the simple mobs
+	name = "jewel of the wastes"
+	desc = "A large blood red gem with metal clasps on it side. An echoing scream can be heard if placed next to the ear"
+	w_class = WEIGHT_CLASS_TINY
+	slot_flags = ITEM_SLOT_EARS
+	icon_state = "jewel_of_wastes"
+	item_state = "jewel_of_wastes"
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+
+/obj/item/clothing/ears/jewel_of_the_wastes/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/wearertargeting/earprotection, list(SLOT_EARS))
+
+/obj/item/clothing/ears/jewel_of_the_wastes/equipped(mob/user, slot)
+	..()
+	user.faction |= "mining"
+
+/obj/item/clothing/head/crown_of_the_wastes //Stuck to head, but makes you the same faction as megafuna
+	name = "crown of the wastes"
+	desc = "Rather simple crown and much to small for The Legion to ware. Has a spot were a jewel would go, and seems to have clamps that are spring loaded..."
+	w_class = WEIGHT_CLASS_TINY
+	body_parts_covered = HEAD
+	slot_flags = ITEM_SLOT_HEAD
+	icon_state = "jewelless_crown"
+	item_state = "jewelledcrown"
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+//No armor no space proofing, ask sci if you want to walk into space with just a crown
+/obj/item/clothing/head/crown_of_the_wastes/equipped(mob/user, slot)
+	..()
+	to_chat(user, "<span class='notice'>The crown's springs go off making it stuck!</span>")
+	user.faction |= "boss"
```

Comment:
Same as the previous but worse because: 
1, it allows the user to effortlessly cheese the other megafauna.
2, it doesn't add the `TRAIT_NODROP` when equipped to the head slot and instead adds it on Initialize, which I doubt is the intention just by looking at the chat message above.

Reply:
When testing the trait no drop when picked up is active. Meaning you have to cut if off to remove or ware it. Intented and all that, like the no drop outfits force you to ware such things

Reply:
Mega funa "cheese" is already done and the crown itself makes you have a weak spot if you do PvP